wa/commands/create.py

```diff
@@ -68,6 +78,75 @@ def execute(self, state, args):
+                                      'name already exists.')
+
+        template_group = self.parser.add_mutually_exclusive_group()
+        template_group.add_argument('-A', '--apk-workload', action='store_true',
+                                    help='Use Apk template. This template allows you to specify ' +
+                                         'an APK file that will be installed and run on the device. You should ' +
+                                         'place the APK file into the workload\'s directory at the same level ' +
+                                         'as the __init__.py.')
+        template_group.add_argument('-r', '--revent-workload', action='store_true',
+                                    help='Use revent template. This template will create the Python Class for ' +
+                                         'workloads that required interacting with a devices gui using revent.'
+                                         'This template that makes no assumptions about the nature of your ' +
+                                         'workload, apart from the fact that you need revent automation.')
+        template_group.add_argument('-R', '--apk-revent', action='store_true',
+                                    help='Use Apk revent template. This template will create the Python Class for ' +
+                                         'workloads that required interacting with a devices gui using revent. It' +
+                                         'also allows you to specify an APK file that will be installed and ran on ' +
+                                         'the device. You should place the APK file into the workload\'s directory ' +
+                                         'at the same level as the __init__.py.')
+        template_group.add_argument('-u', '--uiautomation', action='store_true',
+                                    help='Use UI automation template. This template generates a UI automation ' +
+                                         'Android project as well as the Python class. This template that makes ' +
+                                         'no assumptions about the nature of your workload, apart from the fact ' +
+                                         'that you need UI automation.')
+        template_group.add_argument('-U', '--apk-uiautomation', action='store_true',
+                                    help='Use Apk UI automation template. This template generates a UI automation ' +
+                                         'Android project as well as the Python class. It also allows you to specify' +
+                                         'an APK file that will be installed and ran on the device. You should place ' +
+                                         ' the APK file into the workload\'s directory at the same level as the ' +
+                                         ' __init__.py.')
```

Comment:
Given an increasing number of different workload configurations (and it is entirely possible that we will be adding more in the future), perhaps it might make sense to have a single '--kind'  option that takes one of a set of values as an argument, rather than a different option for each kind.